config/initializers/decidim.rb

```diff
@@ -130,3 +131,14 @@
 
 Rails.application.config.i18n.available_locales = Decidim.available_locales
 Rails.application.config.i18n.default_locale = Decidim.default_locale
+
+module Decidim::Budgets::Groups
```

Comment:
you can just add the new workflow, without monkey patching this:
`Decidim::Budgets::Groups.workflows[:pam20202] = DecidimStaging::BudgetsGroupsWorkflowPam2020`

Reply:
Oh, that's Great 🎉 !!